composer.json

```diff
@@ -17,11 +17,13 @@
         "ext-zip": "*",
         "box/spout": "^3.0",
         "cloudinary/cloudinary_php": "^1.12",
-        "doctrine/doctrine-migrations-bundle": "^1.0",
-        "eightpoints/guzzle-bundle": "^7.4",
+        "composer/package-versions-deprecated": "^1.11.99",
```

Comment:
What this package used for?

Reply:
Not entirely sure. It was there already, I only changed the version restraint.
It has to do with Composer 2 support, and it replaces another package used by our dependencies:
```
$ php composer why composer/package-versions-deprecated
danskernesdigitalebibliotek/ddb-cover-service-imports  dev-develop  requires  composer/package-versions-deprecated (^1.11.99)  
composer/package-versions-deprecated                   1.11.99.4    replaces  ocramius/package-versions (1.11.99)              
doctrine/dbal                                          3.1.4        requires  composer/package-versions-deprecated (^1.11.99)  
doctrine/migrations                                    3.3.2        requires  composer/package-versions-deprecated (^1.8)      
doctrine/orm                                           2.10.2       requires  composer/package-versions-deprecated (^1.8)      
symfony/proxy-manager-bridge                           v5.3.4       requires  composer/package-versions-deprecated (^1.8)      
vimeo/psalm                                            4.12.0       requires  composer/package-versions-deprecated (^1.8.0)  
```